package.json

```diff
@@ -7,10 +7,12 @@
     "@hookform/resolvers": "^3.9.0",
     "@radix-ui/react-accordion": "^1.2.1",
     "@radix-ui/react-checkbox": "^1.1.2",
+    "@radix-ui/react-dialog": "^1.1.2",
```

Comment:
[OPTIONAL] Kindly guide people about the existence of those new packages you installed for this PR so that people can avoid to see an error page when they open it from their localhost.